File: src/Core/Houston.Application/ViewModel/PipelineTriggerViewModels/PipelineTriggerViewModel.cs
namespace Houston.Application.ViewModel.PipelineTriggerViewModels {
	public class PipelineTriggerViewModel {
		public Guid Id { get; set; }

		public Guid PipelineId { get; set; }

		public string SourceGit { get; set; } = null!;

		public bool KeyRevealed { get; set; }
	}
}

File: src/Web/WebAngular/src/app/domain/view-models/pipeline-trigger.view-model.ts
import { PipelineTriggerEventViewModel } from "./pipeline-trigger-event.view-model";

export interface PipelineTriggerViewModel {
	id: string;
	pipelineId: string;
	sourceGit: string;
	keyRevealed: boolean;
	pipelineTriggerEvents: PipelineTriggerEventViewModel[];
}
